HPKE Key Encryption Mode: Wording Improvements (#66)
cose-wg/HPKE: draft-ietf-cose-hpke.md

#### COSE-HPKE Recipient Construction

> Because COSE-HPKE supports header protection, if 'alg' parameter is present, it MUST be in protected bucket, and SHALL be a COSE-HPKE algorithm.

Reviewer: ```suggestion
Because COSE-HPKE supports header protection, if the 'alg' parameter is present, it MUST be in the protected header and MUST be a COSE-HPKE algorithm.
```

The unprotected header MAY contain the kid parameter to identify the static recipient public key the sender used.

> This documents RECOMMENDS the use of the 'kid' parameter
> (or other parameters) to explicitly identify the static recipient public key

Reviewer: ```suggestion
Use of the 'kid' parameter is RECOMMENDED
to explicitly identify the static recipient public key
```
